src/xrEngine/Text_Console.cpp

```diff
@@ -59,17 +59,17 @@ void CTextConsole::CreateConsoleWnd()
     R_ASSERT2(m_hConsoleWnd, "Unable to Create TextConsole Window!");
 };
 //-------------------------------------------------------------------------------------------
-LRESULT CALLBACK TextConsole_LogWndProc(HWND hWnd, UINT uMsg, WPARAM wParam, LPARAM lParam);
+LRESULT CALLBACK TextConsole_LogWndProc(HWND hWnd, unsigned int uMsg, WPARAM wParam, LPARAM lParam);
```

Comment:
Please, return `UINT` for this function.
```suggestion
LRESULT CALLBACK TextConsole_LogWndProc(HWND hWnd, UINT uMsg, WPARAM wParam, LPARAM lParam);
```